NitroxServer/Server.cs

```diff
@@ -137,7 +142,7 @@ public void PauseServer()
             DisablePeriodicSaving();
             world.EventTriggerer.PauseWorldTime();
             world.EventTriggerer.PauseEventTimers();
-            Log.Info("Server has paused");
+            Log.Info("Server has paused, waiting for players");
```

Comment:
Re Hen: Might be worth slightly editing this just to make it even more clear
```suggestion
            Log.Info("Server has paused, waiting for players to connect");
```